src/Core/Content/Media/Aggregate/MediaThumbnail/MediaThumbnailEntity.php

```diff
@@ -48,7 +48,7 @@ public function setHeight(int $height): void
 
     public function getUrl(): string
```

Comment:
recently we did it like this: https://github.com/shopware/shopware/blob/trunk/src/Core/System/SalesChannel/Aggregate/SalesChannelType/SalesChannelTypeEntity.php#L79-L89

Reply:
that is debatable :see_no_evil: :grin: 

one could argue, that a setter should always set a valid value and not `null` if used, on the other hand, it might be necessary to exactly do that, to reset the property :man_shrugging: